### DISCUSSION TITLE
profile choice at startup

### DISCUSSION THREAD

**Discussion by mistyron** [zen-browser/desktop#1924] [General]
Hi,
Is there any way that I can choose the profile when the browser starts? Either similar to chrome where you get a dialog that lets you choose which profile you want the browser toi start with or maybe even just with a cli argument that will point to a particular profile name, anything? If not, I think it should be added!

**Comment by mikedlr** [comment]
flatpak run io.github.zen_browser.zen -P "Test profile 1"

works as long as the profile already exists

flatpak run io.github.zen_browser.zen -P

allows me to select between profiles.

**Comment by mcmerdith** [comment]
Adding "-P" works on windows as well, but it would be nice to see this implemented as a setting instead of having to alter the shortcut. Also a nicer UI for it but that doesn't matter as much

**Comment by scatternoodle** [comment]
I found I only needed to run with -P once, and didn't need to edit the shortcut. 

I ran it via the flatpak CLI once with -P, un-ticked "Use the selected profile automatically at startup" after selecting my profile, and since then it always prompts me to select profile when I launch Zen using the desktop shortcut.
